docs/sources/flow/reference/components/discovery.marathon.md

```diff
@@ -25,26 +25,34 @@ discovery.marathon "LABEL" {
 
 The following arguments are supported:
 
-| Name               | Type           | Description                                                  | Default | Required |
-| ------------------ | -------------- | ------------------------------------------------------------ | ------- | -------- |
-| `servers`          | `list(string)` | List of Marathon servers.                                    |         | yes      |
-| `refresh_interval` | `duration`     | Interval at which to refresh the list of targets.            | `"30s"` | no       |
-| `auth_token`       | `secret`       | Auth token to authenticate with.                             |         | no       |
-| `auth_token_file`  | `string`       | File containing an auth token to authenticate with.          |         | no       |
-| `proxy_url`        | `string`       | HTTP proxy to proxy requests through.                        |         | no       |
-| `follow_redirects` | `bool`         | Whether redirects returned by the server should be followed. | `true`  | no       |
-| `enable_http2`     | `bool`         | Whether HTTP2 is supported for requests.                     | `true`  | no       |
-
-You can provide one of the following arguments for authentication:
-
+Name                     | Type                | Description                                                   | Default | Required
+------------------------ | ------------------- | ------------------------------------------------------------- | ------- | --------
+`servers`                | `list(string)`      | List of Marathon servers.                                     |         | yes
+`refresh_interval`       | `duration`          | Interval at which to refresh the list of targets.             | `"30s"` | no
+`auth_token`             | `secret`            | Auth token to authenticate with.                              |         | no
+`auth_token_file`        | `string`            | File containing an auth token to authenticate with.           |         | no
+`bearer_token_file`      | `string`            | File containing a bearer token to authenticate with.          |         | no
+`bearer_token`           | `secret`            | Bearer token to authenticate with.                            |         | no
```

Comment:
I think that the bearer token is not supported by marathon from what I understand [here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#marathon_sd_config): 
# Optional `Authorization` header configuration.
# NOTE: The current version of DC/OS marathon (v1.11.0) does not support
# standard `Authentication` header, use `auth_token` or `auth_token_file`
# instead.

Reply:
We reached consensus offline to keep the whole HTTPClientConfig wired in on the component and document it. If future versions of marathon begin to support these authentication methods we will not need to update the component.